source/testing-your-html/index.html.md.erb

```diff
@@ -45,7 +47,10 @@ Each object in the `fixtures` list is a different example of the component, wher
 - `name` is the name of the example
 - `options` are the options that generate this example's `html`
 - `html` is the HTML that GOV.UK Frontend outputs with these options
+- `hidden` is `true` if the fixture does not look or behave differently to the other fixtures provided
+
+Omit `hidden` fixtures from the list when you use fixtures for manual or visual regression testing.
```

Comment:
Is "Omit" used in the rest of this content? "Do not include" could be more plain English

Reply:
So…

```suggestion
Do not include `hidden` fixtures when you use fixtures for manual or visual regression testing.
```

?

Reply:
LGTM, thanks